package.json

```diff
@@ -23,11 +23,12 @@
   },
   "main": "./lib/ffi",
   "dependencies": {
+    "any-promise": "^1.1.0",
     "bindings": "~1.2.0",
     "debug": "2",
+    "nan": "2",
     "ref": "1",
-    "ref-struct": "1",
-    "nan": "2"
+    "ref-struct": "1"
```

Comment:
NPM did reordered nan and ref-struct by itself. I can change it back if you'd like, but NPM seems to want it this way.